### DISCUSSION TITLE
Getting Error while " NPM INSTALL "

### DISCUSSION THREAD

**Discussion by SwayamRana808** [HITK-TECH-Community/Community-Website#860] [Q&A]
![Screenshot 2024-05-11 214023](https://github.com/HITK-TECH-Community/Community-Website/assets/122097724/c5b38ae6-6aa5-4059-8f89-5aadfc877cd6)

**Comment by SwayamRana808** [comment]
Hello , I am a GGSOC ' 24 contributor  I want to contribute to this project via issue #799 . but got error while setting up the project can anyone help me to fix this error?

**Comment by Kajol-Kumari** [comment]
Hey,
As the project is kinda old, please make sure you are on node version 14 and then run `npm install`. You can use nod version manager to have multiple versions of node runing on your pc . ref - https://codedamn.com/news/nodejs/nvm-installation-setup-guide

**Reply by SwayamRana808** [reply]
yep! thank you it worked !

**Reply by hemanthkumar-21** [reply]
@Kajol-Kumari  while using node 14 it is showing problem with npm itself. Can you please help. @SwayamRana808 can you please help tell the process how you setup it in your localhost. I am till getting error please help.

![community](https://github.com/HITK-TECH-Community/Community-Website/assets/106808387/69dc967f-5735-4cf9-a641-42b849930a2a)

**Reply by SwayamRana808** [reply]
I have used node v14.19.0 Try with it . May it solve your problem.

**Reply by Kajol-Kumari** [reply]
@Hemu21  I am seeing a few solution suggested https://stackoverflow.com/questions/66458710/npm-err-cb-never-called, try those if it works like clearing cache and installing with `--force` flag

**Reply by hemanthkumar-21** [reply]
Thank you @Kajol-Kumari  @SwayamRana808 It solved with node version V14.19.0. @Kajol-Kumari  please mention complete node version in readme. so that other contributor can easily setup the project. any way @Kajol-Kumari @SwayamRana808  Thank you for helping
